packages/core/src/utils.js

```diff
@@ -364,11 +364,13 @@ export function mergeDefaultsWithFormData(defaults, formData) {
 
 export function getUiOptions(uiSchema) {
   // get all passed options from ui:widget, ui:options, and ui:<optionName>
+  if (!uiSchema) {
```

Comment:
See my comments in #2226 -- I think these changes in this file might not actually be necessary. If they're not necessary, can we remove them?

Reply:
i needed this change for the render tests to pass. I linked core, made the change and they passed